Web/SQL_Injection.md

```diff
@@ -0,0 +1,39 @@
+/*
+Title: SQL Injection
+Description: Injection of SQL commands directly into databse
+*/
+
+- LAST UPDATED DATE: 2015/12/1
+- LAST UPDATED BY: @zaeyx
+
+## Summary
+
+SQL Injection occurs when a unsanitized field takes content from an untrusted source and passes it directly into an SQL query.
+
+## Capabilities and Risk
+
+SQL Injection allows an attacker to execute arbitrary SQL in the context of the web application.  Potentially gaining the ability to read, write, and modify database contents.
+
+## Detection
+
+Detection can be accomplished by whitelisting known database lookups performed by the web application, and alerting if any commands other than the whitelisted ones are executed against the database in the context of the web application.
+
+## Remediation
+
+Correctly sanitize input into the database from any and all untrusted sources.
```

Comment:
Same as the Command Inject, looking for examples on how to do this, what it means on different web application types, libraries and such